src/App.js

```diff
@@ -1,23 +1,17 @@
-import logo from './logo.svg';
-import './App.css';
+import "./App.css";
+import ContentsArea from "./ContentsArea/ContentsArea";
+import Folder from "./Folder/Folder";
+import Footer from "./Footer/Footer";
+import Header from "./Header/Header";
 
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <Header></Header>
+      <Folder />
+      <ContentsArea />
+
+      <Footer></Footer>
```

Comment:
```suggestion
      <Header /r>
      <Folder />
      <ContentsArea />
      <Footer />
```

[중요도: 중간] Header, Footer의 경우도 children 영역에 아무것도 들어가지 않는다면 바로 닫아도 문제 없어보입니다.

Reply:
1a14c7ddfb00854fb586e81305ed21b0da07aa30 넵! 수정하였습니다!